tests/functional/test_matrix_service.py

```diff
@@ -292,17 +321,53 @@ def test_bundle_url(self):
         timeout = int(os.getenv("MATRIX_TEST_TIMEOUT", 300))
         bundle_fqids_url = INPUT_BUNDLE_URL.format(dss_env=self.dss_env)
 
-        self.request_id = self._post_matrix_service_request(
+        self.request_ids = self._post_matrix_service_request(
             bundle_fqids_url=bundle_fqids_url,
             format="loom")
 
         # wait for request to complete
-        WaitFor(self._poll_get_matrix_service_request, self.request_id)\
-            .to_return_value(MatrixRequestStatus.COMPLETE.value, timeout_seconds=timeout)
+        WaitFor(self._poll_all_requests_in_status, self.request_ids, MatrixRequestStatus.COMPLETE.value)\
+            .to_return_value(True, timeout_seconds=timeout)
         bundle_fqids = ['.'.join(el.split('\t')) for el in
                         requests.get(bundle_fqids_url).text.strip().split('\n')[1:]]
 
-        self._analyze_loom_matrix_results(self.request_id, bundle_fqids)
+        self._analyze_loom_matrix_results(self.request_ids["Homo sapiens"], bundle_fqids)
+
+        self.assertIn("Mus musculus", self.request_ids)
+        self.assertEqual(self._retrieve_matrix_location(self.request_ids["Mus musculus"]), "")
+
+    def test_mixed_species(self):
```

Comment:
Is this only tested in v0?

Reply:
Yeah mostly because the functional tests are getting kind of long since we only have the 10x mouse bundles right now.

Reply:
Makes sense